### ISSUE TITLE
WCAG implementation of changes

### ISSUE THREAD

**Issue by sean-eagles** [Canadian-Geospatial-Platform/geo.ca#150]
After the WCAG review changes will have to be implemented from the findings.

**Comment by jaredkinger** [comment]
### To Do List
- [x] Add href to fullscreen ramp button for keyboard navigation
- [x] Add label element to app.geo.ca search widget 
- [x] Add label element to filterable gallery widget 
- [x] Add title attribute to iframe element
- [x] Fix keyboard focus on featured data leaflet maps
- [x] Add missing image alt text
- [x] Adjust heading elements to be in a sequentially-descending order
- [x] Add skip links
